### ISSUE TITLE
What about file storage?

### ISSUE THREAD

**Issue by woutermont** [solid/data-interoperability-panel#240]
According to the specification, all data seems to be stored in Data Registrations according to Shape Trees. This would seem to preclude us from storing and accessing files like we currently can do with/in a Solid RS.

E.g. (how) would it (still) be possible to use a Solid RS as a file server, next to the differently shaped data I have on it?
E.g. (how) would it (still) be possible to serve a (static) website from a Solid RS?

**Comment by elf-pavlik** [comment]
I agree that we should sort out all those issues ASAP. So far sai-js only supports Data Instances that are RDFSources, we need to add support for Non-RDFSources where we describe them via auxiliary resources.
